<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDInfoDialogBase</class>
 <widget class="QWidget" name="CDInfoDialogBase">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
   <item>
    <layout class="QGridLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="5" column="2">
      <widget class="QLabel" name="lb_revision">
       <property name="text">
        <string>Revision:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="KLineEdit" name="m_title">
       <property name="toolTip">
        <string>Use the name of the artist if there is no title.</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lb_category">
       <property name="text">
        <string>&amp;Category:</string>
       </property>
       <property name="buddy">
        <cstring>m_category</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="m_id">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="KLineEdit" name="m_artist">
       <property name="toolTip">
        <string>Write names as &quot;first last&quot;, not &quot;last, first&quot;. Omit any leading &quot;The&quot;.  Use &quot;Various&quot; for compilations.</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lb_artist">
       <property name="text">
        <string>&amp;Artist:</string>
       </property>
       <property name="buddy">
        <cstring>m_artist</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lb_year">
       <property name="text">
        <string>&amp;Year:</string>
       </property>
       <property name="buddy">
        <cstring>m_year</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="lb_genre">
       <property name="text">
        <string>&amp;Genre:</string>
       </property>
       <property name="buddy">
        <cstring>m_genre</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="KComboBox" name="m_category">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Disc Id values must be unique within a category.</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="KIntSpinBox" name="m_year">
       <property name="maximum">
        <number>2100</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QCheckBox" name="m_multiple">
       <property name="text">
        <string>&amp;Multiple artists</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="KLineEdit" name="m_comment"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lb_comment">
       <property name="text">
        <string>Comment:</string>
       </property>
       <property name="buddy">
        <cstring>m_comment</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lb_title">
       <property name="text">
        <string>&amp;Title:</string>
       </property>
       <property name="buddy">
        <cstring>m_title</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="KComboBox" name="m_genre">
       <property name="toolTip">
        <string>Avoid custom values, as they will be written to CDDB as-is.</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="duplicatesEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lb_id">
       <property name="text">
        <string>Disc Id:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="lb_length">
       <property name="text">
        <string>Length:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLabel" name="m_revision"/>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="m_length">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="m_trackList">
     <property name="toolTip">
      <string>For a CD-Extra, set title to &quot;Data&quot;.</string>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>231</width>
         <height>41</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_changeEncoding">
       <property name="text">
        <string>Change Encoding...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lb_playingOrder">
     <property name="text">
      <string>Playing order:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KLineEdit" name="m_playOrder"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_artist</tabstop>
  <tabstop>m_title</tabstop>
  <tabstop>m_comment</tabstop>
  <tabstop>m_year</tabstop>
  <tabstop>m_category</tabstop>
  <tabstop>m_genre</tabstop>
  <tabstop>m_trackList</tabstop>
  <tabstop>m_playOrder</tabstop>
 </tabstops>
 <includes>
  <include location="local">libkcddb/cdinfo.h</include>
  <include location="local">libkcddb/kcddb.h</include>
  <include location="local">libkcddb/categories.h</include>
  <include location="local">libkcddb/genres.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
